pkg/pushsync/pushsync.go

```diff
@@ -470,6 +461,15 @@ func (ps *PushSync) pushPeer(ctx context.Context, peer swarm.Address, ch swarm.C
 		return nil, true, err
 	}
 
+	// if you manage to get a tag, just increment the respective counter
+	t, err := ps.tagger.Get(ch.TagID())
```

Comment:
this is not correct... you should increment the sent count once the chunk is sent.

Reply:
ok, it was a misunderstanding